libraries/FusionData/FusionData.h

```diff
@@ -1,12 +1,13 @@
-#pragma once
+#ifndef FUSIONDATA
```

Comment:
Tradition uses something like FUSIONDATA_H.  That way it doesn't look like some normal constant.